packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -133,36 +153,40 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
       tickFormat: this.props.tickFormat,
     };
     return (
-      <CartesianChart
-        {...this.props}
-        points={this._points}
-        chartType={ChartTypes.VerticalBarChart}
-        xAxisType={this._xAxisType}
-        calloutProps={calloutProps}
-        tickParams={tickParams}
-        {...(this._isHavingLine && { isCalloutForStack: true })}
-        legendBars={legendBars}
-        datasetForXAxisDomain={this._xAxisLabels}
-        barwidth={this._barWidth}
-        focusZoneDirection={FocusZoneDirection.horizontal}
-        customizedCallout={this._getCustomizedCallout()}
-        getmargins={this._getMargins}
-        getGraphData={this._getGraphData}
-        getAxisData={this._getAxisData}
-        onChartMouseLeave={this._handleChartMouseLeave}
-        /* eslint-disable react/jsx-no-bind */
-        // eslint-disable-next-line react/no-children-prop
-        children={(props: IChildProps) => {
-          return (
-            <>
-              <g>{this._bars}</g>
-              {this._isHavingLine && (
-                <g>{this._createLine(props.xScale!, props.yScale!, props.containerHeight, props.containerWidth)}</g>
-              )}
-            </>
-          );
-        }}
-      />
+      <div id={this._idVBC}>
```

Comment:
Can this impact any cascading style coming from parent elements.

Reply:
as discussed offline, rendering the empty div based on a state. Now it should have no impact.